docs/ReleaseNotes.md

```diff
@@ -6,6 +6,18 @@ As the [versioning guide](Versioning.md) details, it cannot always be determined
 
 ## 2.8
 
+### Features
+
+The `FDBRecordStore` can now use the global meta-data key that was added to FoundationDB in version 6.1 to cache state that otherwise must be read from the database at store initialization time. To do so, the user must first configure their record store to have cacheable meta-data by calling `setStoreStateCacheability(true)` on an existing record store. Then they can configure their `FDBDatabase` to use the new `MetaDataVersionStampStoreStateCache` class to cache this state in memory. The record store will then update the global meta-data key whenever the meta-data changes on any record store running on that cluster. The cache will then invalidate any of its entries that were read at older values of the global meta-data key.
```

Comment:
Oh, hm. Setting the cache on the database ensures that all record stores started with contexts started from that database will use the cache. I felt like this was the easiest way of setting the cache, so it was the method I suggested here, but setting it on all record stores with cacheable (or possibly cacheable) meta-data is also allowed. (And the decision as to whether to update the meta-data version is made only by looking at the `cacheable` flag in the store header, not the type of cache, so that it works even if the client hasn't rolled out the cache everywhere yet.)

I've updated the wording on the docs here to try and make that more clear.